src/main/java/frc/robot/commands/OuttakeAmp.java

```diff
@@ -59,6 +59,7 @@ public OuttakeAmp(Arm arm, StorageIndex index, Shooter shooter) {
    * @param drive   The drivetrain
    */
   public OuttakeAmp(Arm arm, StorageIndex index, Shooter shooter, Drivetrain drive) {
+    // TODO: this line kills simulation because the optional may be empty.
     Pose2d ampPose = DriverStation.getAlliance().get() == Alliance.Red ? VisionConstants.RED_AMP_POSE
         : VisionConstants.BLUE_AMP_POSE;
     Pose2d ampPose2 = DriverStation.getAlliance().get() == Alliance.Red ? VisionConstants.RED_AMP_POSE //TODO
```

Comment:
Can you delete these 2 lines? They cause errors and aren't needed. These are now instance variables that are set in the first InstantCommand. These are probably still here because of merge errors, or maybe I forgot to delete them.

Reply:
That is not the subject of this pull request. This is restoring ArmTest; it is not trying to fix bugs in unrelated commands. Keep focused.